ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_paddle_math.py

```diff
@@ -1272,3 +1273,35 @@ def test_paddle_erf(
         on_device=on_device,
         x=x[0],
     )
+
+
+@handle_frontend_test(
+    fn_tree="paddle.inner",
+    dtype_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float"),
+        shape=(2, 2),
```

Comment:
The tests in the other frontends doesn't constrain to (2, 2) tensors, so we could remove this argument.